src/MakesHttpRequests.php

```diff
@@ -72,7 +72,9 @@ protected function request($verb, $uri, array $payload = [])
             empty($payload) ? [] : ['form_params' => $payload]
         );
 
-        if ($response->getStatusCode() != 200) {
+        $statusCode = $response->getStatusCode();
+
+        if ($statusCode < 200 || $statusCode > 299) {
```

Comment:
This change seems unrelated?

Reply:
This was done as the Execute Command call returns a 204 code on success.  Prior to the change, the code was looking for ONLY a 200 response, therefore successful calls were returning an exception (from the handleRequestError() method).

This change allows for any 2xx responses to return successfully, with the response body handed back to the requestor.

Reply:
Gotcha, makes sense.